src/dispatch/plugins/dispatch_slack/views.py

```diff
@@ -155,7 +155,7 @@ def handle_reaction_added_event(
         event_service.log(
             db_session=db_session,
             source=convo_plugin.title,
-            description=message_text,
+            description=f'"{message_text}," said {individual.name}',
```

Comment:
nit: This feels a bit awkward, perhaps we could just do "Author: " instead of "Said"?

Reply:
Fine :)